example/package.json

```diff
@@ -64,9 +66,11 @@
     "@cumulus/queue-pdrs": "2.0.1",
     "@cumulus/sf-sqs-report": "2.0.1",
     "@cumulus/sync-granule": "2.0.1",
-    "@cumulus/test-processing": "2.0.1"
+    "@cumulus/test-processing": "2.0.1",
+    "snyk": "^1.385.0"
```

Comment:
Hmmm, interesting.   Blocking pending ci/team convo.   This is assumedly a requirement for running it, but *why* is it not just a dev dep?